### DISCUSSION TITLE
Do we have Formula support

### DISCUSSION THREAD

**Discussion by irttboopathy** [singerdmx/flutter-quill#168] [Feature Request]
I wanted to have block of strings, that would be treated as one, similar to formula in quilljs. When I wanted to delete, it should be deleted like Embeddable. This can be helpful to include tags, mentions and many other similar things.

But when I try to achieve that, we are always inserting new line before and after the embed node, that makes it have in separate line.

I came across the class called **Block**. I guess we are using for this use-case, but I don't find any document, or dig into the code to find its working.

**Comment by singerdmx** [comment]
No for now

**Comment by irttboopathy** [comment]
Do we have any roadmap for this??

**Comment by singerdmx** [comment]
No
But you are welcome to submit pull request
